mita/openstack.py

```diff
@@ -176,7 +176,7 @@ def create_node(self, **kw):
                 break
             if datetime.datetime.now() - starttime > timeout:
                 logger.info("Failed to bring the node in running state in {timeout}s".format(timeout=timeout))
-                raise NodeErrorState("Failed to bring up the node in Running state " + name)
+                raise NodeErrorState('node {name} is in "{state}" state'.format(name=name, state=state))
```

Comment:
Whoops, this needs to use `new_node_state[0]`, since that's what we get from above. Like so:

```python
raise NodeErrorState('node {name} is in "{state}" state'.format(name=name, state=new_node_state[0]))
```